ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -256,16 +325,28 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
 
     return (
       <Fragment>
-        {error && <ErrorPanel fixed error={error} />}
         <DataTable
           items={sortedWorkerPools}
           headers={headers}
           sortByLabel={sortBy}
           sortDirection={sortDirection}
-          onHeaderClick={this.handleHeaderClick}
-          renderRow={this.renderRow}
+          onHeaderClick={handleHeaderClick}
+          renderRow={renderRow}
           padding="dense"
         />
+        {open && (
+          <DialogAction
+            open={open}
+            onSubmit={handleDeleteClick}
+            onComplete={handleDialogActionComplete}
+            onClose={handleDialogActionClose}
+            onError={handleDialogActionError}
+            error={error}
+            title={title}
+            body={<Typography>{body}</Typography>}
+            confirmText={confirmText}
```

Comment:
STR:
1. Open dialog
2. Try to delete and wait until the error shows up in the dialog
3. Try to delete again

Actual: The error message doesn't move.

Expected: the error is set to null before attempting to delete again. That way the error message goes away for a split second and only shows up again if the action fails. The benefit of doing this is to give feedback to the user that we have cleared this error and that we are trying again.